Car.java

```diff
@@ -0,0 +1,94 @@
+public class Car {
+
+    private String color;
+    private String mark;
+    private int consumption;
+    private double volume;
+    private double mileage;
+
+
+    public Car(String color, String mark, int cons, double vol, double mil) {
+        this.color = color;
+        this.mark = mark;
+        this.consumption = cons;
+        this.volume = vol;
+        this.mileage = mil;
+    }
+
+
+    public void drive(int km) {
+
+        double volumeForTheTrip = consumption * km;

```

Comment:
If you multiply two integers (without dot) - you're getting floating variable (with dot), right? Or it should be int/long type. 
P.S. 'consumption' could be a float (like 7.9) but you're used int.